tests/components/ResultsCount.test.tsx

```diff
@@ -0,0 +1,135 @@
+import { State } from '@yext/answers-headless-react';
+import { render, screen } from '@testing-library/react';
+import { ResultsCount } from '../../src/components/ResultsCount';
+import { mockAnswersState } from '../__utils__/mocks';
+import { RecursivePartial } from '../__utils__/mocks';
+
+const mockedStateUniversalMultiple: RecursivePartial<State> = {
+  searchStatus: {
+    isLoading: false
+  },
+  meta: {
+    searchType: 'universal'
+  },
+  universal: {
+    verticals: [{
+      resultsCount: 2,
+    },
+
+    {
+      resultsCount: 3,
+    }
+    ] }
+};
+
+const mockedStateUniversalSingle: RecursivePartial<State> = {
+  searchStatus: {
+    isLoading: false
+  },
+  meta: {
+    searchType: 'universal'
+  },
+  universal: {
+    verticals: [{
+      resultsCount: 1,
+    }
+    ] }
+};
+
+const mockedStateUniversalNoResult: RecursivePartial<State> = {
+  searchStatus: {
+    isLoading: false
+  },
+  meta: {
+    searchType: 'universal'
+  },
+  universal: {
+    verticals: [{
+      resultsCount: 0,
+    }
+    ] }
+};
+
+const mockedStateVerticalMultiple: Partial<State> = {
+  searchStatus: {
+    isLoading: false
+  },
+  meta: {
+    searchType: 'vertical'
+  },
+  vertical: {
+    resultsCount: 2
+  }
+};
+
+const mockedStateVerticalSingle: Partial<State> = {
+  searchStatus: {
```

Comment:
i'm thinking we could probably do something like:
```
const mockedVerticalState: Partial<State> = {
searchStatus: {
    isLoading: false
  },
  meta: {
    searchType: 'vertical'
  }
}
```
and then spread that into mockedStateVerticalSingle, mockedStateVerticalMultiple, and mockedStateVerticalNoResult so we can avoid duplicating searchStatus and meta for all these states. Similarly for universal ones.
```
const mockedStateVerticalSingle: Partial<State> = {
  ...mockedVerticalState,
  vertical: {
    resultsCount: 1
  }
}
```